src/Jobs/CheckForUpdatesJob.php

```diff
@@ -48,4 +45,21 @@ public function process()
         // mark job as completed
         $this->isComplete = true;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function afterComplete()
+    {
+        // Queue a new job to run in the future
+        $injector = Injector::inst();
+        $queuedJobService = $injector->get(QueuedJobService::class);
+
+        $startAfter = new DateTime(SS_Datetime::now()->getValue());
+        $startAfter->modify('+1 day');
```

Comment:
Can you make the time period a config string (default: `+1 day`)? Also whether to enable this functionality or not should be configurable (default: true)

Reply:
I've added configuration options now.